CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+Please see [doc/docs/project/CHANGELOG.md
```

Comment:
Maybe we're talking past each other. What I'm saying is: +1 to have documentation in the doc/ directory, -1 on having multiple directories inside doc/. When we get to the point of having more than 30 or so files in the doc/ directory, we can start to talk about subdirectories, but it's not relevant yet.